tests/providers/google/cloud/hooks/test_secret_manager_system.py

```diff
@@ -15,18 +15,18 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from unittest import TestCase
+import os
 
 import pytest
 
 from airflow.providers.google.cloud.hooks.secret_manager import SecretsManagerHook
 from tests.providers.google.cloud.utils.gcp_authenticator import GCP_SECRET_MANAGER_KEY
 from tests.test_utils.gcp_system_helpers import GoogleSystemTest, provide_gcp_context
 
-TEST_SECRET_ID = "test-secret"
-TEST_SECRET_VALUE = "test-secret-value"
-TEST_SECRET_VALUE_UPDATED = "test-secret-value-updated"
-TEST_MISSING_SECRET_ID = "test-missing-secret"
+TEST_SECRET_ID = os.environ.get("GCP_SECRET_MANAGER_SECRET_ID", "test-secret")
+TEST_SECRET_VALUE = os.environ.get("GCP_SECRET_MANAGER_SECRET_VALUE", "test-secret-value")
+TEST_SECRET_VALUE_UPDATED = os.environ.get("GCP_SECRET_MANAGER_VALUE_UPDATED", "test-secret-value-updated")
+TEST_MISSING_SECRET_ID = os.environ.get("GCP_SECRET_MANAGER_SECRET_ID", "test-missing-secret")
```

Comment:
should this be GCP_SECRET_MANAGER_MISSING_SECRET_ID?

Reply:
Yes! Thank you @subkanthi